airmozilla/popcorn/renderer.py

```diff
@@ -30,20 +31,32 @@ def render_edit(edit_id):
             os.stat(filepath)[os.stat.st_size] > 0):
         edit.status = PopcornEdit.STATUS_PROCESSING
         edit.save()
-        process_json(edit.data, out=filepath)
-
-    upload_file_name = '%s.webm' % uuid4().hex[:13]
+        process_json(
+            data=edit.data['data'],
+            out=filepath,
+            background_color=edit.data['background']
+        )
+
+    d = datetime.date.today()
```

Comment:
No, use UTC instead. 
E.g. 
```python
prefix = datetime.datetime.utcnow().strftime('%Y/%m/%d/')
upload_file_name = prefix + uuid4().hex[:13] + '.webm
```
Normally I would use `os.path` but this isn't strictly a directory as S3 is odd.